src/OpenTK/Platform/X11/X11GLNative.cs

```diff
@@ -792,13 +861,15 @@ public override void ProcessEvents()
             // Process all pending events
             while (Exists && window != null)
             {
+                //Functions.XNextEvent(window.Display, ref e);
```

Comment:
Why is this commented here? Can we remove it if we're using `XCheckTypedWindowEvent` later?